WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrder.swift

```diff
@@ -16,9 +16,14 @@ struct NewOrder: View {
         .toolbar {
             ToolbarItem(placement: .confirmationAction) {
                 Button(action: {
-                    // TODO-5405 - Add create order action
+                    viewModel.createOrder()
                 }, label: {
-                    Text(Localization.createButton)
+                    if viewModel.isLoading {
+                        ActivityIndicator(isAnimating: .constant(true), style: .medium)
+                            .accentColor(Color(.navigationBarLoadingIndicator))
+                    } else {
+                        Text(Localization.createButton)
+                    }
```

Comment:
Nit: Rather than having an if/else here I think it's cleaner to provide an enum similar to 
https://github.com/woocommerce/woocommerce-ios/blob/7f87d452e380083ff84213fb97b43d0eebaeb9c3/WooCommerce/Classes/ViewRelated/Orders/Order%20Details/Address%20Edit/EditOrderAddressFormViewModel.swift#L231-L234

The reason being that the view has to do none assumptions on what the `else` means and also enable us to unit test that logic!

Reply:
Thanks, this is a much better approach! I updated this in d7ff8b3.